README.md

```diff
@@ -1 +1,67 @@
-# owasp-zap-ui-scan
\ No newline at end of file
+# OWASP ZAP UI Automated Scanning :zap:
+<p align="center"> <br> :exclamation: :exclamation:  <b> ONLY RUN THIS AGAINST APPLICATIONS YOU HAVE PERMISSION TO ATTACK </b> :exclamation: :exclamation: <br><br> </p>
+
+Provides the ability to execute a [Full Scan](https://github.com/zaproxy/zaproxy/wiki/ZAP-Full-Scan]) against a web application using the OWASP ZAP Docker image within a Azure DevOps pipeline. This generates the standard OWASP ZAP Html report and an NUnit test report to publish the results to the pipeline. 
```

Comment:
You could bullet points the output maybe?

 - standard OWASP ZAP html report (with lowercase h)
 - NUnit test report for publishing results to a pipeline

Reply:
Done (Y) Good idea